recipes/juliaup/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "juliaup" %}
+{% set version = "1.5.37" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/JuliaLang/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0ba906ee94b323c9b72bdae27d74039fb0ab3340b33e6d68fda279b72848598d
+
+build:
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - cargo install --locked --root "$PREFIX" --path .  # [unix]
+    - cargo install --locked --root "%LIBRARY_PREFIX%" --path . || goto :error  # [win]
+    - rm -f "${PREFIX}/.crates.toml"  # [unix]
+    - del /F /Q "%LIBRARY_PREFIX%\.crates.toml"  # [win]
+  number: 0
+  missing_dso_whitelist:
+    - '*/libgcc_s.so.1'  # [linux]
```

Comment:
@isuruf I'm not sure what is our policy on this today? Is this one OK?

Reply:
This is a good question... and I personally just did this because I wanted it to pass without thinking further. There is some precedence for this btw: https://cs.github.com/?q=org%3Aconda-forge+libgcc_s+missing_dso_whitelist

Reply:
A more important question for @isuruf is: how do you feel about adding these packages in general? They obviously are harmless in and of themselves, but their whole point is installing julia independently of our own julia... I personally have mixed feelings but concluded why the hell not

Reply:
The `missing_dso_whitelist` should be solved by adding `{{ compiler('c') }}` to `build`.

Reply:
Thanks @xhochy and @ocefpaf